src/utils/getDiscourseNodes.ts

```diff
@@ -3,11 +3,64 @@ import getSubTree from "roamjs-components/util/getSubTree";
 import discourseConfigRef from "./discourseConfigRef";
 import getDiscourseRelations from "./getDiscourseRelations";
 import parseQuery from "./parseQuery";
+import { Condition } from "./types";
 
-export type DiscourseNode = ReturnType<typeof getDiscourseNodes>[number];
+// TODO - only text and type should be required
+export type DiscourseNode = {
+  text: string;
+  type: string;
+  shortcut: string;
+  specification: Condition[];
+  
+  // TODO - wondering if this should be an enum instead
+  // 'relation' | 'default' | 'user'
+  isRelationBacked: boolean; 
+  canvasSettings: {
+    [k: string]: string;
+  };
+  // @deprecated - use specification instead
+  format: string;
+};
 
-const getDiscourseNodes = (relations = getDiscourseRelations()) =>
-  Object.entries(discourseConfigRef.nodes)
+const DEFAULT_NODES: DiscourseNode[] = [
```

Comment:
This is the biggest idea communicated in the PR - we are now adding default discourse nodes representing Block and Page. This brings all sorts of benefits, including allowing users to include Roam's (and in the future, any app's) base primitives into their discourse schema.

For videos, checkout out the `teamcovid` graph, since their graph appears in the video